src/client/routes/login/PasswordLogin.tsx

```diff
@@ -106,7 +105,8 @@ export const PasswordLogin: React.FunctionComponent<Props> = (): JSX.Element =>
 				<TextLink to="/login">Forgot Username / Password?</TextLink>
 				<FlexRow>
 					<TextLink fontSize="1.4rem" color={STRINGS.ACCENT_COLOR} to="/login/create">
-						New User? Create Account <LeftPaddedImg src={arrowIcon} alt="Right Arrow" />
+						New User ? Create Account
```

Comment:
Is that a ternary or just weirdly formatted text?

Reply:
weird text